src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -1062,6 +1061,11 @@ public PSCmdlet CurrentPSCmdlet
 
         public PackageExtractionContext PackageExtractionContext { get; set; }
 
+        protected void FlushBlockingCollection()
+        {
+            ExecutePSScript(null, false);
```

Comment:
This one is nasty. Have to use named args. 
Also add a comment of the method's purpose.
I think the method itself deserves separate implementation rather than "hacking" `ExecutePSScript`. Does it make sense?